nservicebus/gateway/index.md

```diff
@@ -97,3 +97,12 @@ To disable retries in the gateway use the `DisableRetries` setting:
 snippet: GatewayDisableRetriesConfiguration
 
 When retries are disabled, any messages that fail to be sent to another site will be immediately routed to the configured error queue.
+
+
+### Version compatibility
+
+The Gateway component ensures only forward compatibility for one major version. So a message sent by the NServiceBus 3.x Gateway can be understood by the NServiceBus 4.x Gateway, and a message sent by the NServiceBus 4.x Gateway can be understood by NServiceBus.Gateway 1.x (the independent package targeting NServiceBus 5.x).
```

Comment:
We do have some kind of legacy mode 

https://github.com/Particular/NServiceBus.Gateway/blob/da5198f5bc0393965e366b917bbdc22976cb6411/src/NServiceBus.Gateway/HeaderManagement/GatewayIncomingBehavior.cs#L31

not sure how that's supposed to work. @janovesk do you know?